### ISSUE TITLE
host setup details (CUDNN_STATUS_EXECUTION_FAILED)

### ISSUE THREAD

**Issue by nicolaerosia** [phil-bergmann/tracking_wo_bnw#35]
Hello,

I'm trying to test the tracker with `python experiments/scripts/test_tracktor.py` but I always end up with this error
```
ERROR - test_tracktor - Failed after 0:00:14!
Traceback (most recent calls WITHOUT Sacred internals):
  File "experiments/scripts/test_tracktor.py", line 120, in my_main
    tracker.step(frame)
  File "/home/user/va-workspace/tracking_wo_bnw/src/tracktor/tracker.py", line 257, in step
    self.obj_detect.load_image(blob['data'][0], blob['im_info'][0])
  File "/home/user/va-workspace/tracking_wo_bnw/src/tracktor/fpn.py", line 63, in load_image
    c1 = self.RCNN_layer0(self.im_data)
  File "/home/user/miniconda3/envs/tracking_wo_bnw/lib/python3.6/site-packages/torch/nn/modules/module.py", line 357, in __call__
    result = self.forward(*input, **kwargs)
  File "/home/user/miniconda3/envs/tracking_wo_bnw/lib/python3.6/site-packages/torch/nn/modules/container.py", line 67, in forward
    input = module(input)
  File "/home/user/miniconda3/envs/tracking_wo_bnw/lib/python3.6/site-packages/torch/nn/modules/module.py", line 357, in __call__
    result = self.forward(*input, **kwargs)
  File "/home/user/miniconda3/envs/tracking_wo_bnw/lib/python3.6/site-packages/torch/nn/modules/conv.py", line 282, in forward
    self.padding, self.dilation, self.groups)
  File "/home/user/miniconda3/envs/tracking_wo_bnw/lib/python3.6/site-packages/torch/nn/functional.py", line 90, in conv2d
    return f(input, weight, bias)
RuntimeError: CUDNN_STATUS_EXECUTION_FAILED
```

I believe this might be related to the fact that I'm running the latest NVIDIA Driver (`440.36`) which supports CUDA 10.2 on an NVIDIA 2070 RTX SUPER.
I have tried multiple operating systems (Ubuntu 16.04 with conda env for Python 3.6) and Fedora 31 (conda env).

Could you please detail your setup so I can try to match it?

**Comment by nicolaerosia** [comment]
seems to be the same error as in #16. A friend of mine did the exact setup on 1080Ti and it seems to be working just fine.

**Comment by timmeinhardt** [comment]
Can we close this since it is indeed the same as #16. We are working on a Tracktor version for PyTorch 1.3 and newer CUDA versions. We will release it in a couple of days. I hope this resolves all of these issues.

**Comment by nicolaerosia** [comment]
@timmeinhardt thank you, looking forward for the update.

**Comment by nicolaerosia** [comment]
@timmeinhardt fixed on master branch with PyTorch 1.3.1, thanks!
